### PR TITLE
Update gorm 'info' log level translation

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

# TL;DR
By default, setting the gorm log level to 'info' results in all sql being dumped to logs. This is pretty verbose, so this changes settting the gorm log level to 'info' only when the user config is set to 'debug'

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/2324

## Follow-up issue
_NA_

### CODE REVIEW

**Comment by katrogan** [comment]
cc @ckiosidis

**Review by ckiosidis — APPROVED** [review]
LGTM

**Review by EngHabu — APPROVED** [review]

**Comment by ckiosidis** [comment]
@katrogan  @EngHabu any chance this change can make it in the next release of flyteadmin/datacatalog ?
